test/e2e/specs/interactivity/directive-on-document.spec.ts

```diff
@@ -30,17 +30,28 @@ test.describe( 'data-wp-on-document', () => {
 		page,
 	} ) => {
 		const counter = page.getByTestId( 'counter' );
+		const isEventAttached = page.getByTestId( 'isEventAttached' );
 		const visibilityButton = page.getByTestId( 'visibility' );
+
 		await expect( counter ).toHaveText( '0' );
+		await expect( isEventAttached ).toHaveText( 'yes' );
 		await page.keyboard.press( 'ArrowDown' );
 		await expect( counter ).toHaveText( '1' );
+
 		// Remove the element.
 		await visibilityButton.click();
 		// This keyboard press should not increase the counter.
 		await page.keyboard.press( 'ArrowDown' );
+
 		// Add the element back.
 		await visibilityButton.click();
 		await expect( counter ).toHaveText( '1' );
+		await expect( counter ).toHaveText( '1' );
```

Comment:
Duplicated assertions?

Reply:
Oh, true!

@c4rl0sbr4v0 can you fix this in the PR that you make to solve [this](https://github.com/WordPress/gutenberg/pull/58134#issuecomment-1907708231)?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/58181